lib/ts/recipe/thirdpartypasswordless/components/themes/signInUp/thirdPartyOnlySignInUp.tsx

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
Why is this file needed?

Reply:
This needed to be a separate component because it uses the style context as well as providing it and adding the keeping it in `index.ts` made it feel crowded. I just moved `HeaderWithProviderList` out of it as well.